OFEP-provider-client-mapping

```diff
@@ -0,0 +1,33 @@
+## Provider to client mapping
+
+## State: APPROVED
+
+The goal of this OFEP is to describe a way that application authors can selectively determine which provider a given client uses. This means that they should be able to set a provider for a library that is different than what they use in their main app.
```

Comment:
@thiyagu06 this might apply to your situation, though I think in your case it's more about different parts of a large application.